src/tag.test.keyboard.ts

```diff
@@ -0,0 +1,188 @@
+
+    const event = await eventPromise;
+
+    expect(event).toEqual({
+      bubbles: true,
+      cancelable: false,
+      composed: true,
+      type: 'remove',
+    });
+
+    await expect(host).toBeHidden();
+  },
+);
+
+test(
+  'can be removed via Space',
+  { tag: '@keyboard' },
+  async ({ mount, page }) => {
+    await mount(
+      () => html`<glide-core-tag label="Label" removable></glide-core-tag>`,
+    );
+
+    const host = page.locator('glide-core-tag');
+    const button = page.getByRole('button');
+
+    const eventPromise = host.evaluate((element) => {
+      return new Promise((resolve) => {
+        element.addEventListener(
+          'remove',
+          (event) => {
+            resolve({
+              bubbles: event.bubbles,
+              cancelable: event.cancelable,
+              composed: event.composed,
+              type: event.type,
+            });
+          },
+          { once: true },
+        );
+      });
+    });
+
+    await button.focus();
+    await button.press('Space');
+
+    const event = await eventPromise;
+
+    expect(event).toEqual({
+      bubbles: true,
+      cancelable: false,
+      composed: true,
+      type: 'remove',
+    });
+
+    await expect(host).toBeHidden();
+  },
+);
+
+test(
+  'can edit via Enter when editable',
+  { tag: '@keyboard' },
+  async ({ mount, page }) => {
+    await mount(
+      () =>
+        html`<glide-core-tag label="Label" private-editable></glide-core-tag>`,
+    );
+
+    const host = page.locator('glide-core-tag');
+    const button = page.getByRole('button');
+
+    await expect(host).toDispatchEvents(async () => {
+      await button.focus();
+      await button.press('Enter');
+    }, [
+      {
+        bubbles: true,
+        cancelable: false,
+        composed: true,
+        type: 'edit',
+      },
+    ]);
+  },
+);
+
+test(
+  'can edit via Space when editable',
+  { tag: '@keyboard' },
+  async ({ mount, page }) => {
+    await mount(
+      () =>
+        html`<glide-core-tag label="Label" private-editable></glide-core-tag>`,
+    );
+
+    const host = page.locator('glide-core-tag');
+    const button = page.getByRole('button');
+
+    await expect(host).toDispatchEvents(async () => {
+      await button.focus();
+      await button.press('Space');
+    }, [
+      {
+        bubbles: true,
+        cancelable: false,
+        composed: true,
+        type: 'edit',
+      },
+    ]);
+  },
+);
+
+test(
+  'cannot be removed via Enter when disabled',
+  { tag: '@keyboard' },
+  async ({ mount, page }) => {
+    await mount(
+      () =>
+        html`<glide-core-tag
+          label="Label"
+          disabled
+          removable
+        ></glide-core-tag>`,
+    );
+
+    const host = page.locator('glide-core-tag');
+    const button = page.getByRole('button');
+
+    await expect(host).not.toDispatchEvents(async () => {
+      await button.focus();
+      await button.press('Enter');
+    }, [{ type: 'remove' }]);
```

Comment:
Here and elsewhere, no need to focus first, right?

```suggestion
    await expect(host).not.toDispatchEvents(
      () => button.press('Enter'),
      [{ type: 'remove' }],
    );
```

Reply:
Is there an existing test that focuses first and doesn't need to? If so, I'll clean it up. Or feel free to push a commit.

Reply:
Ah, this was taken from a few older tests we had before where we focused. You removed it, if I recall correctly, but I didn't do it from my end. I'll do that now!